src/common/pageGenerator.ts

```diff
@@ -0,0 +1,96 @@
+import { GraphORMBuilder } from './../graphorm/graphorm';
+import { GraphQLResolveInfo } from 'graphql';
+import { Context } from './../Context';
+import { Connection, ConnectionArguments } from 'graphql-relay';
+import graphorm from '../graphorm';
+import { Page, PageGenerator } from '../schema/common';
+import { getCursorFromAfter } from './pagination';
+import { base64 } from './base64';
+
+export interface GQLPage extends Page {
+  timestamp?: Date;
+}
+
+export interface PageGeneratorConfig {
+  maxSize?: number;
+}
+
+export interface PageGeneratorEntity {
+  createdAt: Date;
+}
+
+const DEFAULT_PAGE_SIZE = 300;
+const DEFAULT_MAX_SIZE = 300;
+
+export function createPageGenerator({
```

Comment:
Not sure about the name because it's not an all-purpose page generator. It's a page generator for entities with `createdAt` property. What do you think?

Reply:
Ah, yes. I will think of a much more precise name for this. Thanks for the thought!